backend/apiPackage.ts

```diff
@@ -67,4 +69,31 @@ export async function getPackagesByName(req : Request, res: Response){
   }catch(error){
     return res.status(500).send(`Error in getPackagesByName: ${error}`);
   }
-}
\ No newline at end of file
+}
+
+export async function extractMetadataFromZip(filebuffer: Buffer): Promise<apiSchema.PackageMetadata> {
```

Comment:
{
  "metadata": {
    "Name": "Underscore",
    "Version": "1.0.0",
    "ID": "underscore"
  },
  "data": {
    "Content": "example",
    "JSProgram": "if (process.argv.length === 7) {\nconsole.log('Success')\nprocess.exit(0)\n} else {\nconsole.log('Failed')\nprocess.exit(1)\n}\n"
  }
}

This is what should be returned to the user for /package. So need to return the data and JSProgram. (from spec.yaml file)